### PR TITLE
log update (results for extra models, weighted loss and augmented data)

### PR DESCRIPTION
**Pull request by tanikina**
This adds the last batch of experiments with `dialam2024_merged_relations` config. The current best-performing model is `microsoft/deberta-v3-large`. Below are the evaluation results on the validation set for different settings (including weighted loss, training with only 20 most frequent classes etc.):

| **model**                                                                          | **run**                                                                                                                     | **macro-f1** | **micro-f1** | **args-general-f1** | **args-focused-f1** | **illoc-general-f1** | **illoc-focused-f1** |
|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|--------------|--------------|---------------------|---------------------|----------------------|----------------------|
| BART                                                                               | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/hkl8qxpe) | 0.384        | 0.704        | 0.568               | 0.335               | 0.852                | 0.702                |
| XLNet                                                                              | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/iy6y2bkc) | 0.390        | 0.706        | 0.527               | 0.294               | 0.856                | 0.703                |
| DeBERTa                                                                            | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/9n6r81r2) | 0.428        | 0.712        | 0.589               | 0.361               | 0.846                | 0.697                |
| DeBERTa with weighted loss                              | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/dlpsiqa5) | 0.417        | 0.719        | 0.587               | 0.363               | 0.856                | 0.700                |
| DeBERTa with only frequent classes (20 vs 26 classes in total) | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/q3e65vrr) | 0.432    | **0.724**    | 0.584               | 0.349               | 0.857                | 0.701                |
| DeBERTa with augmented data (paraphrased L-node texts)              | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/jch3qtuv) | 0.403        | 0.711        | 0.564               | 0.335               | **0.859**            | **0.707**            |
| DeBERTa-v3                                                                         | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/z46l0wk9) | 0.431    | **0.723**    | **0.601**           | **0.363**           | 0.851                | 0.704                |
| DeBERTa-v3 original + EDA-augmented data (combined train set)                      | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/df4xituu) | 0.399    | 0.716    | 0.585           | 0.364           | 0.850                | 0.698                |
| DeBERTa-v3 additionally fine-tuned on EDA-augmented data                      | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/ojttbaf3) | 0.398    | 0.715    | 0.571           | 0.332           | 0.856                | 0.700                |
| DeBERTa-v3 with only "officially blacklisted" nodesets                      | [W&B run](https://wandb.ai/tanikina/dialam2024_merged_relations-re_text_classification_with_indices-training/runs/hmige0zt) | **0.449**    | 0.715    | 0.557           | 0.323           | 0.852                | **0.707**                |

### CODE REVIEW

**Comment by ArneBinder** [comment]
# full result on val data:

$ python3 src/evaluation/eval_official.py --gold_dir=data/train --predictions_dir=data/validation_annotated_deberta_v3 --mode=illocutions
general.p: 0.8622892315729807
general.r: 0.847089799405591
general.f1: 0.8511310912354281
focused.p: 0.717489532623461
focused.r: 0.6981894061035216
focused.f1: 0.7034717513399968
 
$ python3 src/evaluation/eval_official.py --gold_dir=data/train --predictions_dir=data/validation_annotated_deberta_v3 --mode=arguments
general.p: 0.6643397895535604
general.r: 0.5891417270425421
general.f1: 0.6005552985712661
focused.p: 0.49017500285485893
focused.r: 0.31741342856450766
focused.f1: 0.36328749501622953
